packages/page-staking/src/Nomination/ControllerAccountSelector.tsx

```diff
@@ -0,0 +1,81 @@
+// Copyright 2017-2020 @polkadot/app-123code authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import {Button, InputAddress} from '@polkadot/react-components';
+import {useTranslation} from '@polkadot/app-accounts/translate';
```

Comment:
No. This needs to be from the staking app.

Reply:
PS: Linting will also catch the spacing here... assuming it does its job properly.